controller/utils.go

```diff
@@ -620,7 +620,7 @@ func getSchedulersAndGlobalPortRanges(
 	for _, scheduler := range schedulers {
 		configYaml, err := models.NewConfigYAML(scheduler.YAML)
 		if err != nil {
-			log.WithError(err).Error("failed to unmarshal scheduler %s", scheduler.Name)
+			log.WithError(err).Error(fmt.Sprintf("failed to unmarshal scheduler %s", scheduler.Name))
```

Comment:
it would be event batter to take advantage of the structured logger:
```
log.WithError(err).WithField("schedulerName", scheduler.Name).Error("failed to unmarshal scheduler")
```